src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -5,17 +5,17 @@ sidebar_order: 3
 
 Source maps can sometimes be tricky to get going. If you’re having trouble, try the following tips.
 
-### Verify release is configured
+### Verify release is configured in your SDK
 
-It's possible to have created a release and not have tagged your errors with the same release. 
+In order for uploaded source maps to be located and applied, the release needs to be created by the CLI or API (and the correct files uploaded with it), and the name of that newly-created release needs to be included in your SDK configuration.
 
-Open up the issue from the Sentry UI and check if the release is configured. If it says "_not configured_" or "_N/A_" next to **Release** on the right hand side of the screen (or if you do not see a `release` tag in the list of tags), you'll need to go back and [tag your errors]({%- link _documentation/workflow/releases.md -%}#tag-errors). If this is properly set up you'll see "Release: my_example_release". 
+To verify this, open up the issue from the Sentry UI and check if the release is configured. If it says "_not configured_" or "_N/A_" next to **Release** on the right hand side of the screen (or if you do not see a `release` tag in the list of tags), you'll need to go back and [tag your errors]({%- link _documentation/workflow/releases.md -%}#tag-errors). If this is properly set up you'll see "Release: my_example_release". 
 
 ### Verify artifacts are uploaded
 
-Once your release is properly configured and issues are tagged, you can then click on the release >> Artifacts (or **Releases >> your specific release >> Artifacts**) to check that they are in fact uploaded to the correct release (or at all).
+Once your release is properly configured and issues are tagged, you can then click on the release >> Artifacts (or **Releases >> your specific release >> Artifacts**) to check that your source maps and the associated files are in fact uploaded to the correct release (or at all).
```

Comment:
Not your change, but still...

"or at all"

If they're not uploaded to the correct release, it doesn't matter that they're uploaded at all, since AFAIK there's no way to change which release an artifact is associated with after the fact, is there? So I’d get rid of this parenthetical.